src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -103,12 +102,6 @@ public GenotypeCalculationArgumentCollection( final GenotypeCalculationArgumentC
     public double STANDARD_CONFIDENCE_FOR_CALLING = 30.0;
```

Comment:
Make this camelCase since it's not actually a constant.  I think it would have reduced some confusion here.